spec/config/valid_urls.yml

```diff
@@ -87,23 +87,14 @@ long:
 
 international:
   - http://www.xn--rksmrgs-5wao1o.nu
-  - http://www.中国政府.政务.cn
```

Comment:
We removed these example since after this PR is merged they will no longer pass validation